src/libpriv/rpmostree-util.h

```diff
@@ -191,3 +199,31 @@ gboolean
 rpmostree_variant_bsearch_str (GVariant   *array,
                                const char *str,
                                int        *out_pos);
+
+/* these are kept here for easier sharing with the client */
+
+typedef enum {
+  RPMOSTREED_AUTOMATIC_UPDATE_POLICY_NONE,
+  RPMOSTREED_AUTOMATIC_UPDATE_POLICY_CHECK,
+  RPMOSTREED_AUTOMATIC_UPDATE_POLICY_DOWNLOAD,
+  RPMOSTREED_AUTOMATIC_UPDATE_POLICY_REBOOT,
+} RpmostreedAutomaticUpdatePolicy;
+
+const char*
+rpmostree_auto_update_policy_to_str (RpmostreedAutomaticUpdatePolicy policy,
+                                     GError **error);
+
+gboolean
+rpmostree_str_to_auto_update_policy (const char *str,
+                                     RpmostreedAutomaticUpdatePolicy *out_policy,
+                                     GError **error);
+
+typedef enum {
+  RPM_OSTREE_UPDATE_MISSING,
+  RPM_OSTREE_UPDATE_DOWNLOADED,
+} RpmOstreeUpdateOnDiskStates;
+
+typedef enum {
+  RPM_OSTREE_PKG_TYPE_BASE,
+  RPM_OSTREE_PKG_TYPE_LAYER,
+} RpmOstreePkgTypes;
```

Comment:
This one feels more like an `rpmostree-core.h` entry?  Though that's a messy topic...I put the refspec type there and have ended up needing to `#include rpmostree-core.h` in a lot more places including the client.  Dunno.  Maybe a new `rpmostree-types.h`?